data/human/human missions.txt

```diff
@@ -4965,3 +4965,164 @@ mission "Riley"
+				decline
+			label "Agree"
+			`    "Fantastic," she says, "my name is Sophie. This is my younger brother," she points at the man to her side. "Hi, I am Liam," he says, "we have been looking for a ride for some time now, we already started to worry we wouldn't get there in time," he tells you.` 
+			choice
+				"You will be surpised how fast my ship flies, my name is <first> <last> by the way."
+				"Yes, a lot of captains are busy these days. Call me Captain <last>."
+			`   You lead them to your spaceship.`
+				accept
+	on complete
+		conversation
+			`When you arrive at New China's spaceport, the siblings head off to the place where their fathers last will will be read out. Liam turns to you. "Captain <last>, I have transferred the payment to your account, would it be okay for you to wait here until we come back? We will need a ride home." You agree, and go to the local bar to kill some time.`
+			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`
+		payment 50000
+
+mission "A dead man's path II"
+	name "Following Footprints"
+	description "Bring Sophie and Liam to Tundra in the Cebalrai System."
+	passengers 2
+	source
+		planet "New China"
+	destination
+		planet "Prime"
+	waypoint "Terminus"
+	waypoint "Aspidiske"
+	waypoint "Gorvi"
+	waypoint "Enif"
+	to offer
+		has "A dead man's path I: done"
+	on offer
+		conversation
+			`They both look very happy when they see you again. "So you have decided to help us," Liam asks`
+			choice
+				"I do not know what you want me to do yet."
+				"Sorry, but I would be stupid to accept a mission I do not know anything about."
+			`"Of course," Sophie says, "my brother is just can not wait to get started. Maybe I should start with who our father was. We do not know much about him. He came home only occasionally, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he travelled further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan of fantasy books, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all of the waypoints?"`
```

Comment:
```suggestion
			`Sophie looks at her brother amusedly. "Of course, my brother just can't wait to get started," She turns back to you. "Maybe I should start with who our father was. We do not know much about him. He came home only occasionally, do not get me wrong, our parents were not in conflict, his passion just did not allow him to stay at one place very long. He was a captain, not unlike you. He was restless to explore even the furthest parts of the galaxy and we believe he travelled further than most human captains. Sometimes he brought small items back home that looked like nothing we know." She pauses for a moment, lost in memories. "Anyways," she continues, "in his testament he told us that he left something behind for us. But because he is a great fan of fantasy books, he said he would not make it easy for us. Instead he left behind a route of waypoints that we should visit before we go to the place where he left something behind. Of course we could go directly to this place, but we want to respect our fathers last will." She looks directly at you now, "And that is why we need your help. Would you help us to get to all of the waypoints?"`
```
I'm going to improve this tomorrow.